apps/ewallet/test/ewallet/web/paginator_test.exs

```diff
@@ -98,25 +116,58 @@ defmodule EWallet.Web.PaginatorTest do
       result = Paginator.paginate_attrs(Account, %{"per_page" => "per page what?"})
       assert {:error, :invalid_parameter, _} = result
     end
+
+    test "returrns :error if given both attrs.page_record_value and attrs.page" do
```

Comment:
I think this need to be changed by allowing them to be able to pass together otherwise, they can't go to the next page.

Reply:
Wouldn't passing a `page` defeat the original purpose of this PR though? 

User using this feature should go to next page by passing another `page_record_value`?

Reply:
I totally missed that point 😂 

Need to fix the assertion in test files too because of `page_record_field` and `page_record_value` need to be returned in pagination object instead of `page`